src/expr.rs

```diff
@@ -605,12 +608,33 @@ fn rewrite_label(label: Option<ast::Ident>) -> String {
     }
 }
 
+fn extract_comment(span: Span,
+                   context: &RewriteContext,
+                   offset: Indent,
+                   width: usize)
+                   -> Option<String> {
+    let comment_str = context.snippet(span);
+    if contains_comment(&comment_str) {
```

Comment:
Currently yes since rewrite_comment formats an empty string (" ") to an empty comment (//). As far as I understand the code it will happily rewrite everything you throw at him into a comment.